test/libsolidity/util/BytesUtils.cpp

```diff
@@ -198,8 +199,7 @@ string BytesUtils::formatString(bytes const& _bytes, size_t _cutOff)
 				if (isprint(v))
 					os << v;
 				else
-					os << "\\x" << setw(2) << setfill('0') << hex << v;
```

Comment:
Those stream modifiers should actually be banned...

Reply:
Agree. Found some other places it is used, will push a PR later.